libs/methodologies/bold/rule-processors/mass-id/src/rewards-distribution/rewards-distribution.helpers.ts

```diff
@@ -0,0 +1,88 @@
+import { isNil } from '@carrot-fndn/shared/helpers';
+import {
+  and,
+  eventLabelIsAnyOf,
+  eventNameIsAnyOf,
+} from '@carrot-fndn/shared/methodologies/bold/predicates';
+import {
+  type Document,
+  DocumentEventName,
+  type MassIdReward,
+  type RewardActorParticipant,
+  RewardsDistributionActorType,
+} from '@carrot-fndn/shared/methodologies/bold/types';
+import BigNumber from 'bignumber.js';
+
+import type {
+  ActorReward,
+  RewardsDistributionActor,
+} from './rewards-distribution.types';
+
+import { REQUIRED_ACTOR_TYPES } from './rewards-distribution.constants';
+
+export const checkIfHaulerActorExists = (
```

Comment:
Just a suggestion which I think is more elegant, but feel free to keep the name as is.
```suggestion
export const isHaulerActorDefined = (
```

Reply:
https://github.com/carrot-foundation/methodology-rules/pull/216/commits/a0355f74a9449c3ada27be052d317e0c5323c459